### PR TITLE
switch to US017 in reporting functions

### PR DESCRIPTION
**Pull request by fbenke-pik**

### CODE REVIEW

**Comment by orichters** [comment]
I have no strong opinion on this, but [IIASA changes](https://github.com/IAMconsortium/legacy-definitions/pull/11/files) all units to `USD_2017`, not `US$2017`, mainly because they realized the $ sign created problems in some scripts, as far as I understood. We might use that as an opportunity to do that as well.

**Review by LaviniaBaumstark — APPROVED** [review]

File: R/reportEmployment.R [hunk 1]
```diff
@@ -41,30 +41,30 @@ reportEmployment <- function(gdx, improvements, multiplier, subtype, shareManf,
...
-     "Tech|Electricity|Nuclear|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Solar|PV|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Solar|CSP|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Wind|Onshore|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Wind|Offshore|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Biomass|Combined Heat and Power w/o CC|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Oil|DOT|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Geothermal|OM Cost|fixed (US$2005/kW/yr)",
-     "Tech|Electricity|Storage|Battery|For PV|OM Cost|fixed (US$2005/kW/yr)",
+     "Tech|Electricity|Coal|Pulverised Coal w/o CC|Capital Costs (U$2017/kW)",
```
Comment (orichters) [inline R/reportEmployment.R:44]: Here you typed `U$` instead of `US$`

Comment (fbenke-pik) [reply R/reportEmployment.R:44]: Ah! Thanks for spotting

**Comment by orichters** [comment]
What about
- https://github.com/fbenke-pik/remind2/blob/update_2024/vignettes/remind_summary.Rmd
- https://github.com/fbenke-pik/remind2/blob/update_2024/R/calcNetTradeValue.R#L11

By the way: What is the purpose of these `calc…` functions? Is anyone using them at all?

**Review by orichters — APPROVED** [review]
Looks good apart from these two tiny comments. Thanks for implementing that!

**Comment by fbenke-pik** [comment]
> What about
> 
>     * https://github.com/fbenke-pik/remind2/blob/update_2024/vignettes/remind_summary.Rmd
> 
>     * https://github.com/fbenke-pik/remind2/blob/update_2024/R/calcNetTradeValue.R#L11
> 
> 
> By the way: What is the purpose of these `calc…` functions? Is anyone using them at all?

I think we can skip these for now. Not sure if the vignette and calc functions are still used, but we could remove stuff like this as part of a general cleanup of remind2 (there are more artifacts that could be removed).

**Pull request merged**